lighthouse-core/gather/gather-runner.js

```diff
@@ -430,13 +430,16 @@ class GatherRunner {
     }
   }
 
+  /**
+   * @return {!ComputedArtifacts}
+   */
   static instantiateComputedArtifacts() {
     const computedArtifacts = {};
     require('fs').readdirSync(path.join(__dirname, 'computed')).forEach(function(file) {
       // Drop `.js` suffix to keep browserify import happy.
       file = file.replace(/\.js$/, '');
       const ArtifactClass = require('./computed/' + file);
-      const artifact = new ArtifactClass();
+      const artifact = new ArtifactClass(computedArtifacts);
```

Comment:
really not a fan of this intertwined computedArtifacts, artifact object, but I guess it works for now